### ISSUE TITLE
Fingerprint / Face unlock not working (app unlock possible)

### ISSUE THREAD

**Issue by Spheanix** [PixelExperience/android-issues#4136]
## Build date

PixelExperience_vayu-13.0-20221119-1021-OFFICIAL

## Expected Behavior

By using the fingerprint to unlock the smartphone in locked status, 
you should get to the home screen (= unlock)

## Current Behavior

When using the fingerprint / face unlock, the smartphone will not unlock. Only by PIN the unlock is possible.

## Possible Solution

None - sorry 😔

## Steps to Reproduce

1. Setting up PIN / face unlock / finger print for unlocking the phone
2. Lock the phone (power button once) and wait a little bit
3. Try to unlock the phone by either fingerprint / face unlock 
4. Unlock not working - PIN has to be used

/device vayu
/version thirteen

**Comment by pixelexperiencebot** [comment]
Issue created! You can close at any time by commenting ```/close```

**Comment by Bagualisson** [comment]
Try reinstall rom and format data, don't flash any module or any kernel.

**Comment by pixelexperiencebot** [comment]
Issue closed by maintainer request.

**Comment by Spheanix** [comment]
> Try reinstall rom and format data, don't flash any module or any kernel.

I did reinstall this Rom 3 times. I used the "older" Rom
PixelExperience_vayu-13.0-20221109-1123-OFFICIAL
as well. 
Even the plus versions for each of the 2 ones - problem is persistent.

The only thing which is confusing:
After doing the sideload, phone is telling me 1,000000, whereas PC showing "Total xfer: 0.98x".
This happened for all of the ROMs.

I unlocked my phone as described in the instructions, wiped the data and flashed the img in Fastboot.

Sorry for the inconvenience... Only looking for a solution regarding this specific case. I like your work and the ROM :-)

Regards
Matthias
